### ISSUE TITLE
"Address family not supported by protocol"

### ISSUE THREAD

**Issue by vitaly-zdanevich** [spacedriveapp/spacedrive#2509]
### Describe the bug

$ ./Spacedrive-linux-x86_64.AppImage 
Running from APPIMAGE
2024-05-26T02:19:32.004931Z  INFO sd_core: core/src/lib.rs:89: Starting core with data directory '/home/vitaly/.local/share/spacedrive'
2024-05-26T02:19:32.044331Z  WARN sd_core::util::version_manager: core/src/util/version_manager.rs:159: Config file for sd_core::node::config::NodeConfig does not exist, trying to create a new one with version -> V3
2024-05-26T02:19:32.273508Z  INFO libp2p_swarm: /home/runner/.cargo/git/checkouts/rust-libp2p-5bf6f5083e224a9a/a005656/swarm/src/lib.rs:368: local_peer_id=12D3KooWSUs27p6Vkk1BVbMCPaCuCyywvChzFjE6Q7hsNyfBdoJr
2024-05-26T02:19:32.362457Z  INFO sd_core::p2p::manager: core/src/p2p/manager.rs:151: Setting quic ipv4 listener to: Some(0)
2024-05-26T02:19:32.410975Z  INFO sd_p2p::p2p: crates/p2p/src/p2p.rs:270: HookEvent::ListenerAddrAdded(ListenerId(0), 0.0.0.0:43171)
2024-05-26T02:19:32.411016Z  INFO sd_core::p2p::manager: core/src/p2p/manager.rs:169: Setting quic ipv6 listener to: Some(0)
2024-05-26T02:19:32.416281Z ERROR sd_core: core/src/lib.rs:239: panicked at /home/runner/work/spacedrive/spacedrive/crates/p2p/src/quic/transport.rs:191:26:
called `Result::unwrap()` on an `Err` value: Os { code: 97, kind: Uncategorized, message: "Address family not supported by protocol" } panic.file="/home/runner/work/spacedrive/spacedrive/crates/p2p/src/quic/transport.rs:191" panic.column=26
fatal runtime error: failed to initiate panic, error 3
Aborted

### Platform and versions

```Shell
Gentoo Linux
```

**Comment by HeavenVolkoff** [comment]
Hi, unfortunately, our `AppImage` releases are deprecated, and the only supported release is `deb` for now. We are currently working on Flatpak releases, which should be ready for 0.4. However, the issue you're reporting appears to be related to IPv6 not being available on your machine. Did you disable it by any chance? Even so, this shouldn't be happening, I will take a look into it.

**Comment by vitaly-zdanevich** [comment]
Yes, I disabled IPv6 on my system - my ISP does not support it, and also in order to have a smaller/faster system (Gentoo Linux).

**Comment by jamiepine** [comment]
Closing due to v2 rewrite. If you encounter this issue in v2, please open a new issue.
